1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+   snippet below.
+
+{% highlight html %}
+...
+<!-- Drawer Content -->
+<paper-menu class="list" attr-for-selected="data-route" selected="{{route}}" on-iron-select="onMenuSelect">
+  <a data-route="home" href="/">
+    <iron-icon icon="home"></iron-icon>
+    <span>Home</span>
+  </a>
+...
+{% endhighlight %}
+
+Each navigation menu item consists of an `a` element with two children: 
+`iron-icon` and `span`.
+
+* `iron-icon` displays a single icon. 
+* `span` displays the text next to the icon.
+   
+1. Add the following new navigation item to the bottom of the menu.
+
+{% highlight html %}
+<a data-route="books" href="/books">
+  <iron-icon icon="book"></iron-icon>
+  <span>Books</span>
+</a>
+{% endhighlight %}
+
+Your menu should now look like the code below (some code has been omitted).
+
+{% highlight html %}
+...
+<!-- Drawer Content -->
+<paper-menu class="list" attr-for-selected="data-route" selected="{{route}}" on-iron-select="onMenuSelect">
+  <a data-route="home" href="/">
+    <iron-icon icon="home"></iron-icon>
+    <span>Home</span>
+  </a>
+  ...
+  <a data-route="contact" href="/contact">
+    <iron-icon icon="mail"></iron-icon>
+    <span>Contact</span>
+  </a>
+  <a data-route="books" href="/books">
+    <iron-icon icon="book"></iron-icon>
+    <span>Books</span>
+  </a>
+</paper-menu>
+...
+{% endhighlight %}
+
+If you view the app now, you should see your new item in the navigation
+menu, but the link does not point to a valid page yet. Proceed to the 
+next section to do that now.
+
+<!-- image with new nav item -->
+
+## Adding content
+
+In the previous section you added a navigation menu item to enable the 
+user to navigate to a new page. Now, you add the content for that new page.
+
+1. Open `app/index.html` and find the main content. The beginning of the main
+   content section is provided in the code snippet below (some code omitted for
+   brevity).
+
+{% highlight html %}
+<div class="content">
+  <iron-pages attr-for-selected="data-route" selected="{{route}}">
+
+    <section data-route="home">
+      <paper-material elevation="1">
+        <my-greeting></my-greeting>
+
+        <p class="paper-font-subhead">You now have:</p>
+        <my-list></my-list>
+        ...
+      </paper-material>
+    </section>
+
+    <section data-route="users">
+      <paper-material elevation="1">
+        <h2 class="paper-font-display2">Users</h2>
+        <p>This is the users section</p>
+        <a href="/users/Rob">Rob</a>
+      </paper-material>
+    </section>
+    ...
+{% endhighlight %}
+
+* Each page of the application is a `section` element. 
+  * The `data-route` attribute is an identifier for the routing system. 
+  You'll set that up for your new page in the next section.
+* The `paper-material` element creates a card which floats on top of the
+  main content area. All text, images, and other content should always
+  be contained within one of these cards.
```

Comment:
Since we're tacking a lot of concepts together here, the user is not going to know whether we're talking about a rule that's specific to Polymer, to PSK, to the specific element we're using, or something else. For `data-route`, you've got that covered. Might expand the other two items.

- Each page of the application is a `<section>` element. The `<iron-pages>` element controls which page is displayed at any given time.

For `<paper-material>` In this case, we're talking about a _design_ constraint. The design of this app is that all the content should be on these cards. It's not required for routing, data binding, or anything like that. "Should always be contained" suggests that it's a requirement of Polymer, or PSK, or `<iron-pages>`. I don't have a specific suggestion here, but I think we should be more clear that this is just a matter of design, and you can use all the rest of PSK and choose to format your content some other way. 

(I'll stop short of editorializing and saying that you should ALWAYS format your content another way :D signed, anonymous card-hater.)